### DISCUSSION TITLE
Welcome to openturbine Discussions!

### DISCUSSION THREAD

**Discussion by michaelasprague** [Exawind/openturbine#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by faisal-bhuiyan** [comment]
Hi 👋 , this is Faisal Bhuiyan. I am a researcher/SW developer at NREL. Looking forward to being heavily involved with the development of OpenTurbine and working with all of you.

**Comment by peter-greaves** [comment]
Hi, Peter Greaves from the Offshore Renewable Energy Catapult in the UK here! Very excited to see where OpenTurbine goes, it looks perfect for being the basis of a next-generation aero-elastic code.
